.github/workflows/verify-this-repo.yml

```diff
@@ -8,3 +8,6 @@ on:
 jobs:
   call-verify-workflow:
     uses: Lombiq/GitHub-Actions/.github/workflows/verify-submodule-pull-request.yml@dev
+  
+  call-check-merge-conflict:
+    uses: Lombiq/GitHub-Actions/.github/workflows/check-merge-conflict.yml@issue/OSOE-144
```

Comment:
> Not to have to add a third workflow to all projects, this should be run as part of the verify-submodule-pull-request workflow and the build ones (since the superprojects like OSOCE don’t use the former).

Reply:
Ok, this is a bit unclear now. 

Added this to the [verify-submodule-pull-request](https://github.com/Lombiq/GitHub-Actions/pull/60/commits/3457cbe24ddccbab9b0a4986dd53b813aeb3d880) workflow

Should this also be added to the [build-and-test-orchard-core](https://github.com/Lombiq/GitHub-Actions/blob/dev/.github/workflows/build-and-test-orchard-core.yml) workflow? As the first step or the last? I think the first would be the most logical.

Reply:
That was my initial idea, but thinking about it, this is not really part of the build (and may potentially need other triggers, see my other comment).

So, please do the following refactoring:

- Similar to `verify-submodule-pull-request`, add a `validate-pull-request` reusable workflow (I'm intentional with the verb change here, the former was initially only meant to verify that the submodule has a corresponding PR in the superproject).
- Keep the new action and use it from both workflows. (No need for a separate workflow just for running the new action; we'll have more similar checks in the new workflow instead.)
- In OSOCE, add a new workflow that only calls `validate-pull-request`.
- Document it here: https://lombiq.atlassian.net/wiki/spaces/DEV/pages/19496967/Creating+a+new+Visual+Studio+project+or+solution#Creating-a-new-solution

Reply:
Added the documentation step to the after review section.